plugins/mux/serve_interceptor.go

```diff
@@ -68,3 +87,16 @@ func (w *writerWrapper) WriteHeader(statusCode int) {
 	w.statusCode = statusCode
 	w.ResponseWriter.WriteHeader(statusCode)
 }
+
+func newWriterWrapperWithHijacker(writer http.ResponseWriter, hijacker http.Hijacker) *writerWrapperWithHijacker {
+	return &writerWrapperWithHijacker{
+		w:        newWriterWrapper(writer),
+		Hijacker: hijacker,
+	}
+}
+
+type writerWrapperWithHijacker struct {
+	w *writerWrapper
+	http.ResponseWriter
```

Comment:
The parent wrapper already contains this interface, please delete the duplicate one.

Reply:
Done.